projects/plugins/boost/package.json

```diff
@@ -0,0 +1,116 @@
+{
+	"name": "jetpack-boost",
+	"version": "1.1.0",
+	"description": "Boost your WordPress site's performance, from the creators of Jetpack",
+	"directories": {
+		"test": "tests"
+	},
+	"dependencies": {
+		"@wordpress/components": "^14.1.2",
+		"@wordpress/element": "^3.1.1",
+		"jetpack-boost-critical-css-gen": "git+ssh://git@github.com/automattic/jetpack-boost-critical-css-gen.git#trunk"
+	},
+	"devDependencies": {
+		"@babel/core": "^7.14.3",
+		"@babel/preset-env": "^7.14.4",
+		"@rollup/plugin-commonjs": "^19.0.0",
+		"@rollup/plugin-node-resolve": "^13.0.0",
+		"@rollup/plugin-typescript": "^8.2.1",
+		"@sveltejs/eslint-config": "github:sveltejs/eslint-config",
+		"@tsconfig/svelte": "^2.0.1",
+		"@typescript-eslint/eslint-plugin": "^4.26.1",
+		"@typescript-eslint/parser": "^4.26.1",
+		"@wordpress/e2e-test-utils": "^5.3.0",
+		"@wordpress/i18n": "^4.1.1",
+		"@wordpress/prettier-config": "^1.0.5",
+		"babel-jest": "^27.0.2",
+		"danger": "^10.6.4",
+		"eslint": "^7.28.0",
+		"eslint-plugin-import": "2.23.4",
+		"eslint-plugin-svelte3": "^3.2.0",
+		"expect-puppeteer": "^5.0.4",
+		"husky": "^7.0.0",
+		"jest": "^27.0.4",
+		"jest-puppeteer": "^5.0.4",
+		"node-sass": "^6.0.0",
+		"node-wp-i18n": "^1.2.5",
+		"npm-run-all": "^4.1.5",
+		"prettier": "npm:wp-prettier@2.2.1-beta-1",
+		"prettier-plugin-svelte": "^2.3.0",
+		"puppeteer": "^9.1.1",
+		"rollup": "^2.51.1",
+		"rollup-plugin-copy": "^3.4.0",
+		"rollup-plugin-css-only": "^3.1.0",
+		"rollup-plugin-scss": "^3.0.0",
+		"rollup-plugin-svelte": "^7.1.0",
+		"rollup-plugin-svelte-svg": "^0.2.3",
+		"rollup-plugin-terser": "^7.0.2",
+		"sass": "^1.34.1",
+		"sass-loader": "^12.0.0",
+		"svelte": "^3.38.2",
+		"svelte-check": "^2.1.0",
+		"svelte-preprocess": "^4.7.3",
+		"tslib": "^2.2.0",
+		"typescript": "^4.3.2"
+	},
+	"scripts": {
```

Comment:
Scripts could also use a look to see what's no longer needed. Prettier, linting, and so on should be done at the monorepo level now, for example.